app/views.py

```diff
@@ -193,8 +178,8 @@ def view_all_lists():
 								'status': 'pass',
 								'message': 'lists found'}), 200
 			return jsonify({'count': '0', 'status': 'fail', 'message': 'no lists found'}), 404
-		return jsonify({"error":"user not in isinstance"}), 401
-	return jsonify({"error":" cant access to login"}), 401
+		return jsonify({"message":"user not in isinstance"}), 401

```

Comment:
again, return message will not help the user know what went wrong

Reply:
ok, managed to fix it